charts/latest/blobfuse-csi-driver/templates/csi-blobfuse-controller.yaml

```diff
@@ -134,8 +134,8 @@ spec:
               name: msi
           resources:
             limits:
-              cpu: 200m
-              memory: 200Mi
+              cpu: 2
+              memory: 2
```

Comment:
If the memory unit is omitted, it will be measured in bytes. I think 2 is not reasonable.

Reply:
it's 2Gi, typo